From 439dd258bb85c6edf37f6dc55b4d27f4bdc9b04c Mon Sep 17 00:00:00 2001
From: Brad Hall <brad@nicira.com>
Date: Thu, 24 Nov 2011 07:44:02 -0800
Subject: [PATCH] Fix for bug 894431

Make sure we set the correct interface id

(cherry picked from commit 94ed3e71ac5ea79fc4746d5d1b0c758744316cc6)

Change-Id: I427284182cc0a5a456b43b777faba0a74f3816e7
---
 nova/network/linux_net.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/nova/network/linux_net.py b/nova/network/linux_net.py
index 2d011dd..a6aea8a 100755
--- a/nova/network/linux_net.py
+++ b/nova/network/linux_net.py
@@ -995,7 +995,7 @@ class LinuxOVSInterfaceDriver(LinuxNetInterfaceDriver):
                         '--', '--may-exist', 'add-port', bridge, dev,
                         '--', 'set', 'Interface', dev, "type=internal",
                         '--', 'set', 'Interface', dev,
-                                "external-ids:iface-id=nova-%s" % dev,
+                                "external-ids:iface-id=%s" % dev,
                         '--', 'set', 'Interface', dev,
                                 "external-ids:iface-status=active",
                         '--', 'set', 'Interface', dev,
-- 
1.7.6.5

